editorSupport/atom-reason/src/AtomReasonOutline.re

```diff
@@ -26,8 +29,7 @@ let makeTokens data => {
     | "Class"
     | "Constructor"
     | "Exn"
-    | "Module"
-    | "Signature" => buildToken "class-name" name
```

Comment:
Oh. I didn't find `Signature` [here](https://github.com/the-lambda-church/merlin/blob/2e014d8a8bd44af25796b292706889c5513599c1/src/frontend/protocol.mli#L66), and thought it was added by mistake. Will bring it back.